src/StatsTools/variational_energy_estimator.jl

```diff
@@ -0,0 +1,76 @@
+"""
+    variational_energy_estimator(shifts, overlaps; skip=0, kwargs...)
+    variational_energy_estimator(df::DataFrame; max_replicas=:all, skip=0, kwargs...)
```

Comment:
`skip` is a keyword for `ratio_of_means` but here it is seems like a keyword for `variational_energy_estimator`

Reply:
You are right. There is potential for confusion. I removed `skip` from the list here.